src/osbuild-manifests/aws_tag.py

```diff
@@ -0,0 +1,79 @@
+import os
+import subprocess
+import json
+import argparse
+
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--stream', dest='stream', type=str, help='Fedora stream', required=True)
+    args = parser.parse_args()
+
+    builds = getBuildsForStream(args.stream)
+    for build in builds:
+        build_id=build['id']
+        arches=build['arches']
+        for arch in arches:
+            print(f"The build is {build_id}")
+            buildFetch(args.stream, build_id, arch)
+            meta = open('builds/'+build_id+'/'+arch+'/meta.json')
+            data = json.load(meta)
+
+            # Delete this when actually running. Just here while I make this script
+            data ={"amis":[{
+                "name": "us-east-1",
+                "hvm": "ami-0016d5df3041499f9",
+                "snapshot": "snap-0c1ca4850fcd5e573"
+            }]}
+            amis = data['amis']
+            for ami in amis:
+                checkAndAddTag(ami["hvm"], ami["name"])
+                checkAndAddTag(ami["snapshot"], ami["name"])
+            return
+
+def checkAndAddTag(resourceId, region):
+    tagExists = checkTag(resourceId)
+    if tagExists:
+        print(f"{resourceId} already tagged with FedoraUser=coreos tag")
+    else:
+        addTag(resourceId, region)
+        print(f"FedoraUser=coreos tag successfully added to {resourceId}")    
+
+def checkTag(resourceId):
+    checkTagCmd = f'aws ec2 describe-tags --filters Name=resource-id,Values={resourceId} Name=value,Values=coreos'
+    try:
+        tagCheck=subprocess.run([checkTagCmd], shell=True, capture_output=True, text=True)
+        if "FedoraUser" and "coreos" in tagCheck.stdout:
+            return True

```

Comment:
would probably be better if we added `--output=json` to the command and then parsed that (i.e. more easily machine readable) rather than doing this (which is kinda like a `grep`).